ethstats/ethstats.go

```diff
@@ -388,15 +376,23 @@ func (s *Service) login(conn *websocket.Conn, sendCh chan *StatsPayload) error {
 		network = fmt.Sprintf("%d", lesInfo.Network)
 		protocol = fmt.Sprintf("les/%d", les.ClientProtocolVersions[0])
 	}
-	if s.eth != nil {
-		etherBase, err = s.eth.Etherbase()
-		if err != nil {
-			return err
+
+	if s.backend.IsProxy() {
+		// Proxy needs a delegate send here to get ACK
```

Comment:
I'm not totally clear on what's happening here

Reply:
a proxy is not able to sign therefore it has to forward all the sign operations to the validator and sign it with the validator signer

Reply:
> Can you complete the PR template? I'm particularly interested on how backwards compatible these changes are

will do

Reply:
If a delegate sign message is sent from the proxy to the validator in the `sendStats` call on line 410, then why is the proxy now waiting for delegate signed msg from the proxied validator before the `sendStats` call?

Reply:
it will wait for the channel in 383 and 388